### ISSUE TITLE
The gvar composite glyph example may be wrong

### ISSUE THREAD

**Issue by wezm** [MicrosoftDocs/typography-issues#1067]
https://learn.microsoft.com/en-us/typography/opentype/spec/gvar#point-numbers-and-processing-for-composite-glyphs goes through an example of applying deltas to a composite glyph. However, I think the tuple variation data selected for the (1, 1) region is incorrect. It shows the following:

> The data for R3 the following deltas:
> 
>   | pt 0 | pt 1 | pt 2 | pt 3
>  | -- | -- | -- | -- | --
> X | 0 | 21 | -6 | 25
> Y | 0 | 0 | 0 | 0

However those points correspond to the (-1, 1) region, which would not be selected by the (0.2, 0.7) variation instance selected for the example. As a result the final deltas calculated in the example don't match what I expect to see.

I considered that it might be a difference in the version of Skia used in the example so I compared the version of Skia included with QuickDraw GX in Mac OS 7.6.1 and macOS 11.7.1 and the regions are the same. This is the relevant TTX dump of the two regions:

```xml
      <tuple>
        <coord axis="wght" value="-1.0"/>
        <coord axis="wdth" value="1.0"/>
        <delta pt="0" x="0" y="0"/>
        <delta pt="1" x="21" y="0"/>
        <delta pt="2" x="-6" y="0"/>
        <delta pt="3" x="25" y="0"/>
      </tuple>
      <tuple>
        <coord axis="wght" value="1.0"/>
        <coord axis="wdth" value="1.0"/>
        <delta pt="0" x="0" y="0"/>
        <delta pt="1" x="-8" y="0"/>
        <delta pt="2" x="-30" y="0"/>
      </tuple>
```

So I think R3 should be using:

  | pt 0 | pt 1 | pt 2 | pt 3
 | -- | -- | -- | -- | --
X | 0 | -8 | -30 | 0
Y | 0 | 0 | 0 | 0

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 446bcdab-0883-6d30-b5ed-0d04262ff820
* Version Independent ID: cdf10ecc-926a-a992-fb71-67213ea8113f
* Content: [gvar — Glyph Variations Table (OpenType 1.9) - Typography](https://learn.microsoft.com/en-us/typography/opentype/spec/gvar#processing-the-gvar-table)
* Content Source: [typographydocs/opentype/spec/gvar.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/opentype/spec/gvar.md)
* Product: **typography**
* GitHub Login: @PeterCon
* Microsoft Alias: **alib**

**Comment by PeterCon** [comment]
See the [OT 1.9.1 alpha](https://docs.microsoft.com/en-us/typography/opentype/otspec191alpha/ot191alpha) for draft revisions addressing this issue.

**Comment by wezm** [comment]
1.9.1 changes look good 👍

**Comment by PeterCon** [comment]
This is fixed in OT 1.9.1. See the [OT 1.9.1 beta](https://docs.microsoft.com/en-us/typography/opentype/otspec191/ot191beta) for draft revisions addressing this issue. Closing.
